src/utils/index.js

```diff
@@ -16,6 +18,18 @@ export const trimArrayToMaxOf6 = array => {
   return array.length <= 6 ? array : array.slice(0, 6)
 }
 
+export const getFontScale = fontScale => {
+  if (Array.isArray(fontScale)) {
+    return trimArrayToMaxOf6(fontScale)
+  }
+
+  if (fontScalePresets.hasOwnProperty(fontScale)) {
+    return fontScalePresets[fontScale]
+  } else {
+    throw new Error(`No Font Scale Preset Found for "${fontScale}", the presets available are: "${Object.keys(fontScalePresets)}"`)
+  }
```

Comment:
I really like this. Especially informing the user in the error of the available presets.